dvc/command/experiments/show.py

```diff
@@ -373,7 +373,7 @@ def prepare_exp_id(kwargs) -> "RichText":
 
 
 def baseline_styler(typ):
-    return {"style": "bold"} if typ == "baseline" else {}
+    return {"style": "bold"} if typ == "baseline" else {"style": "default"}
```

Comment:
hmm, I don't see a "default" style in https://github.com/willmcgugan/rich/blob/2ba277ac54a153d3030b496698133ec2e4f67e1d/rich/default_styles.py. Does this even work?

Reply:
It works but it is was only added to make it easier to set `on {bcolor}`

Reply:
@daavoo, can we remove it now?

Reply:
Yes. I Forgot to do it in #7212